src/app/components/CommentsList.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import Comment from '../components/Comment';
+
+export default class CommentList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comment: [],
```

Comment:
```suggestion
      comments: [],
```
так код бывает легче читать) массив всегда указывает на множественное число, поэтому лучше так писать:)